dev/ci/ci-riscv_coq.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download riscv_coq
+
+( cd "${CI_BUILD_DIR}/riscv_coq"
+  make EXTERNAL_DEPENDENCIES=1 \
+       COQMF_ARGS="-Q ${CI_INSTALL_DIR}/lib/coq/user-contrib/coqutil coqutil"
```

Comment:
Do you have any idea why `coqutils` isn't found if this variable isn't set like this?

Reply:
I don't think I set up the makefile correctly. It doesn't look like jobs which depend on other jobs are actually sharing those artifacts. I added the dependency in the makefile and got rid of this line. Hopefully things will be found now.